launcher/src/main/java/org/wso2/carbon/launcher/Constants.java

```diff
@@ -28,7 +28,8 @@
     public static final String CARBON_START_TIME = "carbon.start.time";
 
     public static final String PROFILE = "profile";
-    public static final String DEFAULT_PROFILE = "profiles/default";
+    public static final String PROFILES = "profiles";
```

Comment:
By default when we create a profile, it is will be created in the osgi folder (<CARBON_HOME>/osgi) but in CARBON 5 we have give the location as <CARBON_HOME>/osgi/profiles/<profile_name> in the pom.xml. Therefore shouldn't profiles here be the path to which the profile is created?